pkg/kubelet/config/file_linux_test.go

```diff
@@ -370,6 +386,10 @@ func expectUpdate(t *testing.T, ch chan interface{}, testCase *testCase) {
 		select {
 		case got := <-ch:
 			update := got.(kubetypes.PodUpdate)
+			if len(update.Pods) == 0 {
```

Comment:
In some test, the watched path doesn't exist at the beginning, so a empty pods will be [reported](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/config/file.go#L92). It is what this code filter for